src/ImageSharp/Image{TPixel}.cs

```diff
@@ -104,6 +139,11 @@ internal Image(Configuration configuration, ImageMetaData metadata, IEnumerable<
         /// </summary>
         public IImageFrameCollection<TPixel> Frames => this.frames;
 
+        /// <summary>
+        /// Gets the clear color to initialize the image frame pixels with.
+        /// </summary>
+        internal TPixel ClearColor => this.clearColor;
```

Comment:
Move this to the `ImageFrame<TPixel>` and add to the constructor there. Use RootFrame as a default for subsequent frames

Reply:
Done, good point there.